### DISCUSSION TITLE
Tidal streams english channel

### DISCUSSION THREAD

**Discussion by lpro-max** [Parcels-code/Parcels#1467] [Q&A]
@willirath @erikvansebille

Update: OK I got it finally loaded the data in a fieldset. Will add a constant velocity of boat, datetime of departure, 
tidal coefficient correction, leeway and CTS (Course to Steer) and animation....

I have done the process in arcgis. [Cross_English_Channel](https://gis.stackexchange.com/questions/458106/crossing-english-channel-in-a-boat/) and in the animation [Animation over time]https://github.com/OceanParcels/parcels/assets/76074770/12f11425-291e-441c-b7f6-7c88eec3529c)
And stumbled on OceanParcels in Linkedin.

[Data in NetCdf format] (https://api.tidetech.org/v1/dataset/english-channel-sample/data/?codes=49,50&file_format=nc)
The sample data is 120 hours long(5 days) (timestamp every 30 minutes). 

![Working](https://github.com/OceanParcels/parcels/assets/76074770/3a31059a-1af7-4d2f-8afc-8cfbcac2a392)

https://github.com/OceanParcels/parcels/assets/76074770/12f11425-291e-441c-b7f6-7c88eec3529c

**Comment by erikvansebille** [comment]
Thanks for reaching out, @lpro-max. I think that @willirath works on a very similar project using Parcels to predict/map ship routes. He may be able to point you in the right direction?
